runtime/src/accounts_db.rs

```diff
@@ -7504,13 +7506,13 @@ impl AccountsDb {
         ignore_mismatch: bool,
         store_hash_raw_data_for_debug: bool,
         enable_rehashing: bool,
+        run_in_background: bool,
     ) -> Result<(), BankHashVerificationError> {
         use BankHashVerificationError::*;
 
         let use_index = false;
         let check_hash = false; // this will not be supported anymore
-                                // interesting to consider this
-        let is_startup = true;
+        let is_startup = !run_in_background;
```

Comment:
nit: We could probably do away with `is_startup` variable, and just pass `run_in_background` directly to `CalcAccountsHashConfig` for `use_bg_thread_pool` field. This would get rid of the negation here and the negation there

Reply:
great point. cleaned up